app/helpers/pairings_helper.rb

```diff
@@ -46,22 +46,55 @@ def set_side_button(player, side, pairing)
   end
 
   def presets(pairing)
-    return [
-      { score1_corp: 3, score2_runner: 0, score1_runner: 3, score2_corp: 0, label: '6-0' },
-      { score1_corp: 3, score2_runner: 0, score1_runner: 0, score2_corp: 3, label: '3-3 (C)' },
-      { score1_corp: 0, score2_runner: 3, score1_runner: 3, score2_corp: 0, label: '3-3 (R)' },
-      { score1_corp: 0, score2_runner: 3, score1_runner: 0, score2_corp: 3, label: '0-6' }
-    ] unless pairing.stage.single_sided?
-
-    return [
-      { score1_corp: 3, score2_runner: 0, score1_runner: 0, score2_corp: 0, label: '3-0' },
-      { score1_corp: 0, score2_runner: 3, score1_runner: 0, score2_corp: 0, label: '0-3' }
-    ] if pairing.side.try(:to_sym) == :player1_is_corp
-
-    return [
-      { score1_corp: 0, score2_runner: 0, score1_runner: 3, score2_corp: 0, label: '3-0' },
-      { score1_corp: 0, score2_runner: 0, score1_runner: 0, score2_corp: 3, label: '0-3' }
-    ] if pairing.side.try(:to_sym) == :player1_is_runner
+    # Double-sided round
+    unless pairing.stage.single_sided?
+      return [
+        { score1_corp: 3, score2_runner: 0, score1_runner: 3, score2_corp: 0, label: '6-0' },
+        { score1_corp: 3, score2_runner: 0, score1_runner: 0, score2_corp: 3, label: '3-3 (C)' },
+        { score1_corp: 0, score2_runner: 3, score1_runner: 3, score2_corp: 0, label: '3-3 (R)' },
+        { score1_corp: 0, score2_runner: 3, score1_runner: 0, score2_corp: 3, label: '0-6' }
+      ]
+    end
+
+    # Single-sided swiss round
+    if pairing.round.stage.format == :single_sided_swiss.to_s
```

Comment:
I think this could call `pairing.stage.single_sided_swiss?`. I don't understand why all the other if statements are using `try(:to_sym)` instead of just `pairing.player1_is_corp?` and `pairing.player1_is_runner?` though?

Reply:
Cleaned this up.